lib/octokit/client/repositories.rb

```diff
@@ -480,9 +480,24 @@ def tags(repo, options = {})
         paginate "#{Repository.path repo}/tags", options
       end
 
-      # List branches
+      # List all topics for a repo
       #
       # Requires authenticated client for private repos.
+      # @param repo [Integer, String, Hash, Repositroy] A GitHub repository.
+      # @return [Array<Sawyer::Resource>] Array of hashes representing topics.
+      # @see https://developer.github.com/v3/repos/#list-all-topics-for-a-repository
+      # @example
+      #   Octokit.topics('octokit/octokit.rb')
+      # @example
+      #   @client.topics('octokit/octokit.rb')
```

Comment:
Totally minor copy/pasta thing—the example is duplicated.

Reply:
Whoops! I'll update in a moment